docs/userGuide/tweakingThePageStructure.md

```diff
@@ -148,20 +147,43 @@ If you wish insert scripts at the bottom, before MarkBind's scripts, simply inse
 
 ---
 
-### Fixing the header to the top
+### Sticking the header to the top
 
-Headers are commonly included inside the HTML `<header>` tag. In encouraging this, a convenient interface to implement <tooltip content="Headers that stick to the top of the page while scrolling the content">fixed headers</tooltip> surrounding the `<header>` tag is provided that ensures page anchors work correctly.
+A sticky header can be implemented by simply adding the `sticky` attribute to a `<header>` element.
 
-****To fix the `<header>`****
-1. Add the `fixed` attribute to your `<header>` element in the layout per the above example.
+Using this attribute as opposed to setting `position: sticky` manually in your stylesheets comes with several conveniences:
+- MarkBind will take care of **aligning the viewport with page anchors** when scrolled to.
```

Comment:
Is this point about not letting elements be covered by the header when clicking on anchor links? Perhaps can reword it to state just that? I feel that a bit confusing to say it in a current way, what do you think?